cmd/policy/policy_test.go

```diff
@@ -1330,13 +1331,31 @@ func TestTestRunner(t *testing.T) {
 				assert.Contains(t, s, "<?xml")
 			},
 		},
+		{
```

Comment:
Test suggestion:
`compile: true`, but owner-id isn't provided.

Reply:
That is the current test case

Reply:
My understanding is if compilation is required, but owner-id isn't provided, it should fail in some way.
But the expectation is pass in this test?

Reply:
Owner-ID is only required if you need to pull a private orb